components/browser/engine-gecko/src/main/java/mozilla/components/browser/engine/gecko/GeckoEngineSession.kt

```diff
@@ -8,6 +8,9 @@ import mozilla.components.concept.engine.EngineSession
 import org.mozilla.geckoview.GeckoRuntime
 import org.mozilla.geckoview.GeckoSession
 
+const val PROGRESS_START = 25
+const val PROGRESS_STOP = 100
```

Comment:
We should file a bug and ask the GeckoView team to provide us with percentages. Even if it won't be as nice as WebView.. they should be responsible for defining this. And I hope we get at least something that's better than this two-step progress.